UnitTests/GitUI.Tests/UserControls/RevisionGrid/Graph/LaneInfoProviderTests.cs

```diff
@@ -234,7 +234,7 @@ public void GetLaneInfo_should_display_the_subject_if_singleline_body_null()
             _realCommitNode.GitRevision.HasMultiLineMessage = false;
             _laneNodeLocator.FindPrevNode(Arg.Any<int>(), Arg.Any<int>()).Returns(x => (_realCommitNode, isAtNode: false));
 
-            GetLaneInfo_should_display(_realCommitNode, suffix: _realCommitNode.GitRevision.Subject);
+            GetLaneInfo_should_display(_realCommitNode);
```

Comment:
No, the `LaneInfoProvider` must be adapted.

Reply:
Implementation in `LaneInfoProvider()` is OK. 
The difference now is that even if Body is null, Subject is used if `HasMultiLineMessage == false`

Reply:
I will check myself when MS VS works again.

Reply:
You are right.